### ISSUE TITLE
.desktop file includes nodisplay

### ISSUE THREAD

**Issue by danrabbit** [mubitosh/qrshare#20]
Your app needs to be launchable from the applications menu. Setting `nodisplay` means it will not appear

**Comment by mubitosh** [comment]
Since the app gets the file name to share after selection only, it is not useful to have it launchable from the applications menu.
If it is indeed a constraint to have it launchable from the applications menu for this case also, then I'll have to modify the app.
Please check the screenshots below showing the use case:

![one](https://user-images.githubusercontent.com/5291239/29747498-8510ee92-8b1a-11e7-946b-08f474d7d1bd.png)
![two](https://user-images.githubusercontent.com/5291239/29747499-912d8244-8b1a-11e7-9b44-2297e1cfcc63.png)
![three](https://user-images.githubusercontent.com/5291239/29747501-95c316c0-8b1a-11e7-8fbc-03fa396ec42c.png)

**Comment by danrabbit** [comment]
Yes, please modify your app so that it shows something useful when launched from the applications menu. This could be as simple as showing a Granite.Welcome widget with a button that opens the file chooser dialog

**Comment by mubitosh** [comment]
Sounds good. Will update the app to behave in that fashion.

**Comment by mubitosh** [comment]
Cooking up something like this
![screenshot from 2017-08-29 09-30-27](https://user-images.githubusercontent.com/5291239/29803940-be8f38f8-8c9c-11e7-8e72-02a5731238c8.png)
